src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  color: #3d4e61;
+  font-family: Avenir-book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  letter-spacing: 0;
+  text-align: left;
+  font-weight: 300;
+}
+
+.big__container {
+  margin: 20px 0;
+}
+
+.big__search {
+  height: 70px;
+  padding-left: 62px;
+  top: 0;
```

Comment:
I believe `top: 0;` doesn't work here, because you don't have position for big__search
you can remove `top: 0`